sklearn/model_selection/tests/test_search.py

```diff
@@ -440,6 +440,7 @@ def test_grid_search_when_param_grid_includes_range():
 
 
 def test_grid_search_bad_param_grid():
+    X_, y_ = make_classification(n_samples=200, n_features=100, random_state=0)
```

Comment:
why not call those variables X and y directly?

Reply:
Done